docs/content/en/docs/workshops/packages/prometheus/prometheus_grafana.md

```diff
@@ -0,0 +1,200 @@
+
+#### Change prometheus-server global configs
+
+By default, `prometheus-server` is configured with `evaluation_interval`: `1m`, `scrape_interval`: `1m`, `scrape_timeout`: `10s`. Those values can be overwritten if preferred / needed.
+
+The following config allows the user to do such customization:
+```yaml
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  name: generated-prometheus
+  namespace: eksa-packages-<cluster-name>
+spec:
+  packageName: prometheus
+  server:
+    global:
+      evaluation_interval: "30s"
+      scrape_interval: "30s"
+      scrape_timeout: "15s"
+```
+
+#### Run prometheus-server as statefulSets
+
+By default, `prometheus-server` is created as a deployment with `replicaCount` equals to `1`. If there is a need to increase the replicaCount greater than `1`, a user should deploy `prometheus-server` as a statefulSet instead. This allows multiple `prometheus-server` pods to share the same data storage.
+
+The following config allows the user to do such customization:
+```yaml
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  name: generated-prometheus
+  namespace: eksa-packages-<cluster-name>
+spec:
+  packageName: prometheus
+  server:
+    replicaCount: 2
+    statefulSet:
+      enabled: true
+```
+
+#### Disable prometheus-server and use node-exporter only
+
+A user may disable the `prometheus-server` when:
+- they would like to use node-exporter to expose hardware- and kernel-related metrics, while
+- they have deployed another metrics collector in the cluster and configured a remote-write storage solution, which fullfills the prometheus-server functionality (check out the [ADOT with Amazon Managed Prometheus and Amazon Managed Grafana workshop]({{< relref "../adot/adot_amp_amg.md" >}}) to learn how to do so).
+
+The following config allows the user to do such customization:
+```yaml
+    apiVersion: packages.eks.amazonaws.com/v1alpha1
+    kind: Package
+    metadata:
+      name: generated-prometheus
+      namespace: eksa-packages-<cluster-name>
+    spec:
+      packageName: prometheus
+      server:
+        enabled: false
+```
+
+#### Disable node-exporter and use prometheus-server only
+
+A user may disable the node-exporter when:
+- they would like to deploy multiple prometheus-server packages for a cluster, while
+- deploy only one or none node-exporter instance per node.
+
+The following config allows the user to do such customization:
+```yaml
+    apiVersion: packages.eks.amazonaws.com/v1alpha1
+    kind: Package
+    metadata:
+      name: generated-prometheus
+      namespace: eksa-packages-<cluster-name>
+    spec:
+      packageName: prometheus
+      nodeExporter:
+        enabled: false
+```
+
+### Prometheus Package Test
+To ensure the Prometheus package is installed correctly in the cluster, a user can perform the following tests.
+
+#### Access prometheus-server web UI
+
+Port forward Grafana to local host `9090`:
```

Comment:
```suggestion
Port forward Prometheus to local host `9090`:
```
Is that what you mean, or am I misunderstanding?

Reply:
You are right. Great observation!